examples/amazon-pay-button-example/README.md

```diff
@@ -0,0 +1,188 @@
+The example shows how to embed Amazon Pay button even without fully integrated backend (see 😇 **_Bypassing a plugin_** chapter in the bottom to check what it is about).
+
+## Features
+
+<!-- Highlight some of the features your module provide here -->
+
+- ⛰ &nbsp;`amazon-pay` plugin to add amazon pay script to the DOM
+- 🚠 &nbsp;`AmazonPayButton.client.vue` component to use in a Vue project
+- 🌅 &nbsp;`useAmazonPayButton` composable function to manage a button mounting
+- 🛤️ &nbsp;`useAmazonPayCheckout` composable function to process the checkout flow
+
+## Requirements
+
+- A basic knowledge of [Amazon Pay](https://developer.amazon.com/docs/amazon-pay-checkout/introduction.html) official documentation - thanks to this, you will understand the further configuration and potential issues
+- [Amazon Pay App](https://store.shopware.com/en/swag117522576433f/amazon-pay.html) installed in your shop
+- Some Nuxt.js project
+
+
+## Setup
+
+### Amazon
+
+Prepare your Amazon Pay integration, you will need:
+- storeId
+- publicKeyId
+- private key (do not version it!)
+- merchantId
+
+Also, there will be a step to define return URLs like: `checkoutReviewReturnUrl` or `checkoutResultReturnUrl`. That URLs should match the routes in your nuxt application as you will be redirected there to proceed the amazon pay flow (redirection with `amazonCheckoutSessionId` query param).
+
+
+### Backend
+
+- Make sure that Amazon Pay payment method is enabled and available in your sales channel.
+
+- Set up [Client Credentials](https://shopware.stoplight.io/docs/admin-api/8e1d78252fa6f-authentication#client-credentials) grant type for Admin API communication - **read and update rights for order transactions** 
+
+
+### Frontend: Nuxt 3 project
+
+0. Install the dependencies
+
+   run `pnpm i` command.
+
+   Note that if you are starting from scratches (not using this example as it is) you will probably need as minimum:
+   - [@shopware-pwa/nuxt3-module](https://www.npmjs.com/package/@shopware-pwa/nuxt3-module) - sets up your project to be Shopware 6 aware
+   - [@shopware-pwa/composables-next](https://www.npmjs.com/package/@shopware-pwa/composables-next) - installs all the dependencies for the logic layer
+   - [@shopware/api-client](https://www.npmjs.com/package/@shopware/api-client) - API client for your REST API Shopware 6 backend
+   - [@amazonpay/amazon-pay-api-sdk-nodejs](https://www.npmjs.com/package/@amazonpay/amazon-pay-api-sdk-nodejs) - nodejs client for Amazon Pay API
+
+1. Register the module in your Nuxt 3 project
+
+```js
+  // ./playground/nuxt.config.ts
+  modules: ["@shopware-pwa/nuxt3-module", "../src/module"],
+  // see that "../src/module" points to this dir (from ./playground/nuxt.config.ts file)
+```
+
+2. Setup nuxt project
+
+```ts
+// ./nuxt.config.ts
+export default defineNuxtConfig({
+  extends: ["@shopware-pwa/composables-next/nuxt-layer"],
+  modules: ["@shopware-pwa/nuxt3-module", "../src/module"], // path to amazon-pay module
+  amazonPay: {
```

Comment:
shouldn't it be in the runtimeConfig?

Reply:
the value from module's config is copied to the runtime config in the module.